internal/cmd/beta/sqlserverflex/user/describe/describe_test.go

```diff
@@ -0,0 +1,233 @@
+package describe
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"github.com/google/uuid"
+	"github.com/stackitcloud/stackit-sdk-go/services/sqlserverflex"
+)
+
+var projectIdFlag = globalflags.ProjectIdFlag
+
+type testCtxKey struct{}
+
+var testCtx = context.WithValue(context.Background(), testCtxKey{}, "foo")
+var testClient = &sqlserverflex.APIClient{}
+var testProjectId = uuid.NewString()
+var testInstanceId = uuid.NewString()
+var testUserId = uuid.NewString()
```

Comment:
This userId is not a UUID, but I guess for testing it doesn't really matter

Reply:
True, edited it anyway